js/jquery.validation.js

```diff
@@ -154,10 +177,10 @@
             if (! this[type]) {
                 // Message area doesn't exist so create it.
                 var messageDiv = $('<div></div>').addClass(this.options.messages[type].classes)
-                var messageContent = $('<span></span>').text(message).addClass('message-content');
-                var messageClose = $('<span></span>').addClass('icon icon--cross');
+                var messageContent = $('<span></span>').html(message).addClass('message-content');
+                var messageClose = $('<span></span>').html('&times;').addClass('icon icon--cross');
```

Comment:
That would be closer to what we've got on MLA. Something along the lines of this...

http://codepen.io/achisholm/pen/qdVabr